packages/core/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@aerogearservices/core",
   "version": "1.0.0",
   "description": "Core SDK for AeroGear services",
-  "main": "dist/index.js",
+  "main": "dist/src/index.js",
```

Comment:
ping @wtrocki 

Looks like the typescript compiler puts the resulting file into a src folder instead of creating a bundle in `/dist`. Is this intentional?